tests/schemas/cy/test_language.json

```diff
@@ -17,6 +17,15 @@
             "type": "string"
         }
     ],
+    "hub": {
+        "enabled": true,
+        "submission": {
+            "button": "Testun cyflwyno",
+            "guidance": "Canllawiau cyflwyno",
+            "title": "Arddangosir y teitl ar y canolbwynt pan fydd wedi'i gwblhau",
```

Comment:
```suggestion
            "title": "Arddangosir y teitl ar y canolbwynt pan fydd wedi’i gwblhau",
```

Reply:
@ajmaddaford I can't quite remember what was agreed, but did we say that we would keep the `test_langauge` schemas in here and runner in sync?. If so, we should update the runner one, but obv that would break tests as is.

Reply:
Yes, they should be kept in sync.